### DISCUSSION TITLE
vertical align items in a VStack

### DISCUSSION THREAD

**Discussion by dcsan** [chakra-ui/chakra-ui#4897] [Q&A]
I'm not sure if this is a bug or I'm just using this wrong. It's a really simple request.

I have a two column layout and want to align all the items in each stack at the top. 

Instead they just center...

I'm using this markup:

```
 <HStack verticalAlign='top' height='100%' position='relative' className='debug-blue'>
    <VStack spacing={2} width={marginSize} height={marginHeight} className='sidebar debug-red'>
        {listWallets()}
    </VStack>

    <VStack align='flex-start' alignItems='flex-start' verticalAlign='top' >
        {/* {showPieces()} */}
        {dummyBlocks()}
    </VStack>

</HStack>

```

this is the problem line:

`<VStack align='flex-start' alignItems='flex-start' verticalAlign='top' >`

I have no idea what's the right markup so I just tried everything 😱 

but see this image where the items in the blue box get VCentered.

I got the items at right to align with a `top: 0` CSS tag, which seems like a real hack.

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/qEhNo.png

**Comment by primos63** [comment]
Maybe something like [this](https://codesandbox.io/s/mystifying-bash-wbhwy?file=/src/App.js)?

You can use `Flex` instead of `HStack`. Not sure why you are using `className`. It's not needed.

**Comment by dcsan** [comment]
hey thanks, so I need to add a parent flexbox, vstack won't display flex? it's weird its defaulting to vertical center with no way to control that...

the className is just for commonly used classes, like a `debug` to show borders. Whats the chakra-way to have reusable settings? I read a bit about layers, but somewhat new to all this...

**Comment by pragmaticpat** [comment]
@dcsan - FWIW - I was able to achieve vertical alignment using an `HStack` with `alignItems="start"`

```
<HStack
        spacing={8}
        alignItems="start" // HERE
        borderColor={"black"}
        borderWidth={1}
      >
```
Had you tried that?
